node/forest_libp2p/bitswap/src/request_manager.rs

```diff
@@ -31,6 +33,8 @@ pub struct BitswapRequestManager {
 }
 
 impl BitswapRequestManager {
+    /// A receiver channel of the outbound `bitswap` network events that the [BitswapRequestManager] emits.
+    /// This channel needs to be properly handled to make `get_block` working
```

Comment:
Perhaps it'd be worth elaborating on what `properly` precisely means.

Reply:
Fixed